### PR TITLE
nimbus.sepolia: host era1 files

### PR DESCRIPTION
**Pull request by siddarthkay**
The PR sets up hosting for era1 files for sepolia.
Available here -> https://sepolia.era1.nimbus.team/

status: changes applied

### CODE REVIEW

File: ansible/era1.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  tasks:
+    - local_action: command ./roles.py --check
+      changed_when: false
+
+- name: Configure ERA1 files hosting
+  become: true
+  hosts:
+    - linux-01.ih-eu-mda1.nimbus.sepolia
+  vars:
+    open_ports_default_comment: 'Nginx'
```
Comment (apentori) [inline ansible/era1.yml:17]: it would be clearer if those variable are in the `host_vars` file

File: era1.tf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* ERA1 Files hosting */
+
+resource "cloudflare_record" "era1_sepolia" {
```
Comment (apentori) [inline era1.tf:3]: Shouldn't this be part of `sepolia.tf` ? to have all sepolia terraform resource in the same place ?

Comment (siddarthkay) [reply era1.tf:3]: ah sure yeah I can move it there.

File: ansible/era1.yml (hunk 1 above)
Comment (siddarthkay) [reply ansible/era1.yml:17]: Okay will change.

Comment (siddarthkay) [reply ansible/era1.yml]: done.

File: era1.tf (hunk 2 above)
Comment (siddarthkay) [reply era1.tf]: done.

**Review by apentori — APPROVED** [review]

File: ansible/host_vars/geth-01.ih-eu-mda1.nimbus.holesky.yml [hunk 3]
```diff
@@ -17,6 +17,11 @@ redirect_ports:
     dst: '{{ test_api_beacon_node_rest_port }}'
     comment: 'Test Beacon API ({{ test_api_beacon_node_branch }})'
 
+# Open Nginx Ports
+open_ports_default_comment: 'Nginx'
+open_ports_default_chain: 'SERVICES'
+open_ports_list: [ { port: 80 }, { port: 443 } ]
```
Comment (jakubgs) [inline ansible/host_vars/geth-01.ih-eu-mda1.nimbus.holesky.yml:23]: You don't need this, you're overriding the values set here this way, which is bad:
https://github.com/status-im/infra-nimbus/blob/0c26a3a8f1fc20ae25378fa4d2153e172173078b/ansible/group_vars/nimbus.holesky.yml#L156-L163

File: ansible/host_vars/linux-01.ih-eu-mda1.nimbus.sepolia.yml [hunk 4]
```diff
@@ -21,6 +21,15 @@ redirect_ports:
...
 era_files_path: '/data/era'
 
+# Era1 files hosting
+era1_files_domain: 'sepolia.era1.nimbus.team'
+era1_files_path: '/data/era1'
+
+# Open Nginx Ports
+open_ports_default_comment: 'Nginx'
+open_ports_default_chain: 'SERVICES'
+open_ports_list: [ { port: 80 }, { port: 443 } ]
```
Comment (jakubgs) [inline ansible/host_vars/linux-01.ih-eu-mda1.nimbus.sepolia.yml:31]: You don't need this, you're overriding the values set here this way, which is bad:
https://github.com/status-im/infra-nimbus/blob/0c26a3a8f1fc20ae25378fa4d2153e172173078b/ansible/group_vars/nimbus.sepolia.yml#L111-L122

File: ansible/host_vars/linux-03.ih-eu-mda1.nimbus.mainnet.yml [hunk 5]
```diff
@@ -2,6 +2,11 @@
 era_files_domain: 'mainnet.era.nimbus.team'
 era_files_path: '/data/era'
 
+# Open Nginx Ports
+open_ports_default_comment: 'Nginx'
+open_ports_default_chain: 'SERVICES'
+open_ports_list: [ { port: 80 }, { port: 443 } ]
```
Comment (jakubgs) [inline ansible/host_vars/linux-03.ih-eu-mda1.nimbus.mainnet.yml:8]: Please remove this. Look at `group_vars`.

**Review by jakubgs — CHANGES_REQUESTED** [review]

File: ansible/host_vars/geth-01.ih-eu-mda1.nimbus.holesky.yml (hunk 3 above)
Comment (siddarthkay) [reply ansible/host_vars/geth-01.ih-eu-mda1.nimbus.holesky.yml]: Sorry, Thanks for explaining.

File: ansible/host_vars/linux-03.ih-eu-mda1.nimbus.mainnet.yml (hunk 5 above)
Comment (siddarthkay) [reply ansible/host_vars/linux-03.ih-eu-mda1.nimbus.mainnet.yml]: done.

File: ansible/group_vars/nimbus.sepolia.yml [hunk 6]
```diff
@@ -108,6 +108,7 @@ nimbus_era_files_db_path: '/data/beacon-node-{{ beacon_node_network }}-unstable/
 nimbus_era_files_nclidb_path: '/data/beacon-node-{{ beacon_node_network }}-unstable/repo/build/ncli_db'
 
 # Open Ports
+open_ports_default_comment: 'Nimbus REST API'
```
Comment (jakubgs) [inline ansible/group_vars/nimbus.sepolia.yml:111]: This will have no effect since all rules in `open_ports_list` have comments.

File: ansible/era1.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  hosts: all
+  tags: always
+  become: false
+  run_once: true
+  gather_facts: false
+  tasks:
+    - local_action: command ./roles.py --check
+      changed_when: false
+
+- name: Configure ERA1 files hosting
```
Comment (jakubgs) [inline ansible/era1.yml:12]: You don't need this file. It's exactly the same as `ansible/era.yml` file.

Remember that you're configuring `nginx_sites` for the whole host for one run of `infra-role-nginx`, you don't need multiple playbooks to run the same thing.

File: ansible/host_vars/linux-01.ih-eu-mda1.nimbus.sepolia.yml [hunk 8]
```diff
@@ -29,7 +33,7 @@ origin_certs:
 
 nginx_sites:
   era_files:
-    - listen 80 default_server
```
Comment (jakubgs) [inline ansible/host_vars/linux-01.ih-eu-mda1.nimbus.sepolia.yml:32]: Why was this removed? Was it causing issues?

**Review by jakubgs — CHANGES_REQUESTED** [review]
Almost there.

File: ansible/host_vars/linux-01.ih-eu-mda1.nimbus.sepolia.yml (hunk 8 above)
Comment (siddarthkay) [reply ansible/host_vars/linux-01.ih-eu-mda1.nimbus.sepolia.yml:32]: yeah `ansible` would complain of port 80 being already configured. I don't have the exact error since this executed probably a week ago

Comment (jakubgs) [reply ansible/host_vars/linux-01.ih-eu-mda1.nimbus.sepolia.yml:32]: Fair enough.

File: ansible/era1.yml (hunk 7 above)
Comment (siddarthkay) [reply ansible/era1.yml:12]: Yes.

File: ansible/group_vars/nimbus.sepolia.yml (hunk 6 above)
Comment (siddarthkay) [reply ansible/group_vars/nimbus.sepolia.yml]: Removed.

**Review by jakubgs — APPROVED** [review]
LGTM.
